db/schema.rb

```diff
@@ -177,21 +178,19 @@
     t.text "address1"
     t.integer "provider_id", default: 0, null: false
     t.integer "region_code"
+    t.datetime "created_at", default: -> { "timezone('utc'::text, now())" }, null: false
+    t.datetime "updated_at", default: -> { "timezone('utc'::text, now())" }, null: false
     t.index ["provider_id", "code"], name: "IX_site_provider_id_code", unique: true
   end
 
-  create_table "subject", id: :integer, force: :cascade do |t|
+  create_table "subject", id: :integer, default: -> { "nextval('\"UcasSubjects_Id_seq\"'::regclass)" }, force: :cascade do |t|
     t.text "subject_name"
     t.text "subject_code", null: false
     t.index ["subject_code"], name: "AK_ucas_subject_subject_code", unique: true
     t.index ["subject_code"], name: "IX_subject_subject_code", unique: true
   end
 
-  create_table "temp_tbl", id: false, force: :cascade do |t|
-    t.text "code"
-  end
-
-  create_table "user", id: :integer, force: :cascade do |t|
+  create_table "user", id: :integer, default: -> { "nextval('mc_user_id_seq'::regclass)" }, force: :cascade do |t|
```

Comment:
Does this run for you @tim. Running `rake db:schema:load` failed for me with these default values.

Reply:
I'll have a look at this, but I think @timabell may have pushed this to the wrong PR ... just checking what's up now.